runtime/rasdump/javadump.cpp

```diff
@@ -1484,6 +1484,94 @@ JavaCoreDumpWriter::writeEnvUserArgsHelper(J9VMInitArgs *vmArgs)
 				_OutputStream.writeCharacters("\n");
 			}
 		}
+
+		/* Ignored options for -Xjit and -Xaot follows these rules:
+		 * 1. If -Xjit is followed by -Xint or -Xnojit,
+		 *    then any -Xjit:optionString beforehand appears as ignored.
+		 * 2. If XX:+MergeCompilerOptions is present,
+		 *    then all -Xjit options after -Xint or -Xnojit should not be ignored.
+		 * 3. If multiple -Xjit strings appear, all but the last one will appear as ignored.
+		 *
+		 * Similar rules apply for -Xaot.
+		 */
+
+		jint numXjit = 0;
+		jint numXaot = 0;
+		bool hasXXMerge = false;
+
+		/* These indices ensure all the options before them will be printed as ignored. */
+		jint ignoreXjitBefore = 0;
+		jint ignoreXaotBefore = 0;
+
+		for (jint i = 0; i < args->nOptions; i++) {
+			const char *optionString = args->options[i].optionString;
+
+			if (0 == strncmp(optionString, "-Xint", 6)) {
+				/* Case 1: ignores all options before. */
+				ignoreXjitBefore = numXjit;
+				ignoreXaotBefore = numXaot;
+			} else if (0 == strncmp(optionString, "-Xnojit", 8)) {
+				/* Case 1: ignore all jit options before. */
+				ignoreXjitBefore = numXjit;
+			} else if (0 == strncmp(optionString, "-Xnoaot", 8)) {
+				/* Case 1: ignore all aot options before. */
+				ignoreXaotBefore = numXaot;
+			} else if (0 == strncmp(optionString, "-XX:+MergeCompilerOptions", 26)) {
+				/* Case 2 */
```

Comment:
Missing period; also lines 1527, 1530.

Reply:
I want to note that this does break the existing formatting style of the file, which does not use period (e.g. line 1471). I hope this will not be an issue.

Reply:
I consider line 1471 to contain a "sentence fragment" (it neither starts with a capital nor ends with a period). This comment starts with a capital so it should have the matching period. This comment could be written as a fragment as well. So either
```c
				/* Case 2. */
```
or
```c
				/* case 2 */
```
but not
```c
				/* Case 2 */
```
or
```c
				/* case 2. */
```